pom.xml

```diff
@@ -154,9 +157,9 @@
 
   <repositories>
     <repository>
-      <id>sonatype-oss-snapshots</id>
```

Comment:
this area is incorrect.  The original goes away, snapshots is all that uses that.